WooCommerce/UITestsFoundation/Screens/Login/LoginCheckMagicLinkScreen.swift

```diff
@@ -1,31 +1,34 @@
+import ScreenObject
 import XCTest
 
-private struct ElementStringIDs {
-    static let passwordOption = "Use Password"
-    static let mailButton = "Open Mail Button"
-}
+public final class LoginCheckMagicLinkScreen: ScreenObject {
 
-final class LoginCheckMagicLinkScreen: BaseScreen {
-    private let passwordOption: XCUIElement
-    private let mailButton: XCUIElement
-    private let mailAlert: XCUIElement
+    private let passwordButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["Use Password"]
+    }
 
-    init() {
-        let app = XCUIApplication()
-        passwordOption = app.buttons[ElementStringIDs.passwordOption]
-        mailButton = app.buttons[ElementStringIDs.mailButton]
-        mailAlert = app.alerts.element(boundBy: 0)
+    private let mailButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["Open Mail Button"]
+    }
```

Comment:
It looks like this element is not used any longer. It was used in the local `isLoaded`, but since you're using the one from `ScreenObject` now, it can be removed:

```suggestion
```